libs/langchain/langchain/chat_loaders/imessage.py

```diff
@@ -17,9 +17,7 @@
 from langchain import schema
 from langchain.chat_loaders import base as chat_loaders
 
-if TYPE_CHECKING:
-    import sqlite3
-
+import sqlite3
```

Comment:
We should import this within the method (lazy import) so that it isn't a required dep of the package i think

Reply:
> We should import this within the method (lazy import) so that it isn't a required dep of the package i think

Ya, figured that may be the case.